src/ProjectNeon/Assets/Data/Templates/Team.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Team : Target
+{
+    [SerializeField] public List<Member> members;

```

Comment:
We must treat abstract classes like interfaces. There must be no exposed public members that are not in the abstract class.

We are only using abstract classes instead of interfaces, since Unity doesn't work nicely with interfaces in the editor, and with it's serialization model.

Reply:
@SilasReinagel Removed in favor of `Target.members` method